source/services/ApplicationVersionService.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from "../helpers/ApiRequest";
+
+import VERSION_STATUS from "../types/VersionStatusTypes";
+
+interface GetApplicationVersionStatus {
+  status: VERSION_STATUS;
+  updateUrl: string;
+}
+const getApplicationVersionStatus = async (): Promise<
+  GetApplicationVersionStatus | Record<string, unknown>
```

Comment:
This return type seems ambiguous. Why is `Record<string, unknown>` needed?

Reply:
The return type is also wrong since the api returns an object with string values, hence `Record<string, string>` is more correct.

By using my type instead, we do not need to use `String(updateUrl || '')` in `AppCompabilityContext`

Reply:
But this doesn't end up well if we want to change the fallback return type to `null`